### PR TITLE
Hotfix: Do not specify a separate service account

### PR DESCRIPTION
**Pull request by DavidHuber-NOAA**
# Description
Separate service accounts are not required on any system and causes issues when running CI.  This removes service accounts from the workflow setup scripts.

# Type of change
- Bug fix (fixes something broken)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO

# How has this been tested?
Ran setup scripts on Hera with `ACCOUNT=nems`; verified `account` was set to `nems` in the resulting rocoto XML for all jobs.

# Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes

### CODE REVIEW

**Review by aerorahul — DISMISSED** [review]
lgtm

**Comment by DavidHuber-NOAA** [comment]
I'm going to add a fix to the CI tests as well to do a shallow submodule checkout.  The unit tests are failing due to too much disk space usage.

**Review by aerorahul — DISMISSED** [review]
lgtm

**Comment by DavidHuber-NOAA** [comment]
@CoryMartin-NOAA @RussTreadon-NOAA @danholdaway It appears that the download error I am seeing here and in the Jenkins checkouts is not due to disk limitation, but limitations placed by git-lfs repositories.  The `jcb` repo has too much data in it to enable a recursive checkout from the global-workflow.

See `/scratch1/NCEPDEV/global/David.Huber/GW/gw_test/.git/modules/sorc/gdas.cd/modules/sorc/jcb/lfs/logs/20240719T171340.919113479.log` for a detailed message.
See [this SO thread](https://stackoverflow.com/questions/62905325/this-repository-is-over-its-data-quota-account-responsible-for-lfs-bandwidth-sh) for a possible solution.

**Comment by aerorahul** [comment]
We can set `export GIT_LFS_SKIP_SMUDGE=1` before the `git clone` command (I have no idea how to do that in the Jenkins SCM)
This will allow the cloning of JCB without the LFS data
Before:
```bash
❯❯❯ git clone emcgh:jcb                                                                                                              ✘ 130
Cloning into 'jcb'...
remote: Enumerating objects: 656, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 656 (delta 0), reused 0 (delta 0), pack-reused 638
Receiving objects: 100% (656/656), 120.28 KiB | 8.59 MiB/s, done.
Resolving deltas: 100% (286/286), done.
Downloading etc/jcb-text.png (11 KB)
Error downloading object: etc/jcb-text.png (fdb18dd): Smudge error: Error downloading etc/jcb-text.png (fdb18ddd36bb2285c6e588ad94559babc318eba3e802b9e1c349fed18a3a82af): batch response: This repository is over its data quota. Account responsible for LFS bandwidth should purchase more data packs to restore access.

Errors logged to '/Users/rmahajan/scratch/jcb/.git/lfs/logs/20240719T142102.976203.log'.
Use `git lfs logs last` to view the log.
error: external filter 'git-lfs filter-process' failed
fatal: etc/jcb-text.png: smudge filter lfs failed
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```
After setting `export GIT_LFS_SKIP_SMUDGE=1`,
```bash
❯❯❯ git clone emcgh:jcb
Cloning into 'jcb'...
remote: Enumerating objects: 656, done.
remote: Counting objects: 100% (18/18), done.
remote: Compressing objects: 100% (18/18), done.
remote: Total 656 (delta 0), reused 0 (delta 0), pack-reused 638
Receiving objects: 100% (656/656), 120.28 KiB | 8.59 MiB/s, done.
Resolving deltas: 100% (286/286), done.

Identity set to Rahul Mahajan <aerorahul@users.noreply.github.com>
```

**Comment by DavidHuber-NOAA** [comment]
Alright, I have applied the fix to this PR as well as #2775 and on my local Jenkins clone on Hera.  I then restarted CI testing on #2775, which is now running smoothly.  I **_think_** Hera is now running smoothly, but I will keep an eye on Jenkins before launching another test case.

**Comment by DavidHuber-NOAA** [comment]
Unfortunately, #2775 CI failed to find the `github` Python module.  More investigation underway.

File: ci/scripts/clone-build_ci.sh [hunk 1]
```diff
@@ -44,6 +44,7 @@ if [[ -d global-workflow ]]; then
   rm -Rf global-workflow
 fi
 
+export GIT_LFS_SKIP_SMUDGE=1
```
Comment (aerorahul) [inline ci/scripts/clone-build_ci.sh]: Should not need this after the hotfix, right?

Comment (DavidHuber-NOAA) [reply ci/scripts/clone-build_ci.sh]: Correct.  I'll revert this change here, in my local Jenkins clones, and in #2775 and restart CI in the latter.

**Review by aerorahul — APPROVED** [review]
lgtm

**Comment by DavidHuber-NOAA** [comment]
Closing, merged into #2775.

**Pull request closed without merging**